src/resources/auth/auth.controller.ts

```diff
@@ -45,14 +45,16 @@ export const signup: RequestHandler = async (req, res, next) => {
 		logger.debug('Save user: ', user.email)
 
 		// Send an email to user, containing the activation link
-		const activeUrl = `${req.headers.host}/auth/active/${user.resetToken}`
+		const activeUrl = `http://${req.headers.host}/auth/active/${
```

Comment:
Does req.headers.host contains http/https? In production, we may and should use https